reactome-jenkins-release-config-template.properties

```diff
@@ -0,0 +1,54 @@
+releaseNumber=
+
+pathToSpeciesConfig=
+pathToOrthopairs=
+pathToWebsiteFilesUpdateFolder=
+pathToSkipList=
+dateOfRelease=yyyy-mm-dd
+
+pathToSpeciesConfig=
+pantherFileURL=ftp://ftp.pantherdb.org/ortholog/current_release/
+pantherQfOFilename=QfO_Genome_Orthologs.tar.gz
+pantherHCOPFilename=Orthologs_HCOP.tar.gz
+MGIFileURL=http://www.informatics.jax.org/downloads/reports/HGNC_homologene.rpt
+RGDFileURL=ftp://ftp.rgd.mcw.edu/pub/data_release/GENES_RAT.txt
+XenbaseFileURL=ftp://ftp.xenbase.org/pub/GenePageReports/GenePageEnsemblModelMapping.txt
+ZFINFileURL=https://zfin.org/downloads/ensembl_1_to_1.txt
+
+brenda.password=
+orphanet.user=
+cosmic.user=
+brenda.user=
+orphanet.password=
+cosmic.password=
+dockblaster.password=
+dockblaster.user=
+
+absoluteReleaseDirectoryPath=
+releaseDownloadDirectoryPath=
```

Comment:
Based on the Slack conversation yesterday, can/should this go in the code and make use of `absoluteReleaseDirectoryPath` for the path to prefix the download directory path?

Reply:
I've made that change now